src/auth/index.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+
+const API_URL = 'http://localhost:3000/'
+const LOGIN_URL = API_URL + 'auth/login'
+
+export default {
+
+  user: {
+    authenticated: false
+  },
+
+  login (context, creds, redirect) {
+    axios.get(LOGIN_URL)
+    .then(response => {
```

Comment:
I guess if the login credentials are not valid, this will fall to the catch (line 18). Have you checked this?

Reply:
@taliaga This is still incomplete. The goal will be to make it work with oauth2, meaning it will go through Fiware Lab to get a token, and if it fails to retrieve it then the authentication fails